Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+
+def globals = [XC32_URL: 'http://ww1.microchip.com/downloads/en/DeviceDoc/xc32-v1.42-full-install-linux-installer.run',
+               XC32_PATH: '/opt/microchip/xc32/v1.42/bin',
+               XC32_PATH: '/opt/microchip/xc32/v1.42/bin',
+               CONTIKI: '$HOME/test contiki',
+               CONTIKI_REPO: 'https://github.com/CreatorDev/contiki.git'
```

Comment:
You won't need this anywhere in the code here. the checkout scm picks it up based on the repo being used.